### PR TITLE
feat(deps): update @intlify dependencies to v9.3.0-beta.19

### PR DESCRIPTION
**Pull request by dargmuesli**
Please change this PR's base branch to a newly created `beta` branch :raised_hands: 

To resolve https://github.com/nuxt-modules/i18n/issues/2098

### CODE REVIEW

**Comment by dargmuesli** [comment]
Now that https://github.com/intlify/eslint-plugin-vue-i18n/pull/377 is merged, this one can be used to create a `beta` release as well, it seems :) @ota-meshi

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```
Comment (kazupon) [inline pnpm-lock.yaml:1]: Is it downgrade?  🤔

**Comment by kazupon** [comment]
@dargmuesli 
I think this PR is good since we can resolve https://github.com/nuxt-modules/i18n/issues/2098 issue

File: pnpm-lock.yaml (hunk 1 above)
Comment (dargmuesli) [reply pnpm-lock.yaml:1]: pnpm v8.6.2 reverted the lockfile update seemingly: https://github.com/pnpm/pnpm/releases/tag/v8.6.2
interesting decision for sure, I was confused by that right now as well.

**Comment by kazupon** [comment]
I think we release as v3.0.0-beta, because deps has beta version

**Comment by dargmuesli** [comment]
That's a good idea!

**Comment by ota-meshi** [comment]
Thank you for this PR. The changes look good, but I have a question. Why do we need a `beta` branch?

**Comment by kazupon** [comment]
@ota-meshi 
>Why do we need a beta branch?

I mean we can release beta version on `master` branch!

**Comment by dargmuesli** [comment]
If it's possible to release a beta version from the master branch as well, you can for sure do so!

In other projects I worked on I'm used to be required to add a beta branch for beta releases, but that's just me.

**Comment by ota-meshi** [comment]
Thank you for sharing your thoughts! I believe we can release a beta version from `master` branch.

**Comment by ota-meshi** [comment]
I'm trying to do a beta version release, but `shipjs` doesn't work. For some reason I get an error with the new syntax and `node:` protocol that should be available in Nodejs v18 🤔

**Comment by dargmuesli** [comment]
Yeah, weird. Seems it's not using node?

**Comment by ota-meshi** [comment]
The `esm` package seems to be the cause of the error. It uses the old parser.

https://www.npmjs.com/package/esm
https://github.com/algolia/shipjs/blob/82d001f5a4ad23cde7efebb3b1d450229f050563/packages/shipjs/bin/shipjs#L3

I got around it by replacing it with one that uses esbuild.

https://github.com/intlify/eslint-plugin-vue-i18n/blob/master/shipjs-cli.cjs

We may need to replace shipjs with something else 😓

**Comment by kazupon** [comment]
I feel that shipjs is mostly in the maintenance phase and ESM support is almost never going to happen in the future.

I'm not using shipjs for releases and changelog generation with vue-i18n, nuxt i18n module and others.
When I first started using shipjs, automatic changelog generation was not supported in github releases.
We should try to replace it with another tool.

I use a tool called `bumpp` for releases now.
https://github.com/antfu/bumpp

I use `bumpp` for releases, and GitHub Releases for changelog generation. However, GitHub Releases does not support the file `CHANGELOG.md`.

I want to support file changelog generation as well, so I created my own tool called `gh-changelogen`.
https://github.com/kazupon/gh-changelogen

This tool is supposed to work with github actions. `bumpp` releases, github actions can detect git tags, so I can use github actions to commit the file `CHANGELOG.md` using github releases and sync it with the file `CHANGELOG.md`. and then commit the file `CHANGELOG.md` and sync it.

There are also other tools that the Vue community has created to generate changelogs, such as

- https://github.com/antfu/changelogithub
- https://github.com/unjs/changelogen

We may want to consider these for `eslint-plugin-vue-i18n`. 😉

**Comment by dargmuesli** [comment]
Great overview, thanks kazu! :heart: Would you try what fits best for this project, @ota-meshi? As I don't have write permissions on this repo, I'd have a hard time to do a release.

**Comment by dargmuesli** [comment]
Oh, I just saw you already did a release :partying_face: I'll try it out rn!

**Comment by ota-meshi** [comment]
I often use `changeset` for my package releases.
https://github.com/changesets/changesets
It is also used by svelte and stylelint.
However, in order for this to work well, we need to introduce changeset-bot.
https://github.com/apps/changeset-bot
What do you think about using `changeset` for the releases of this package?

**Comment by kazupon** [comment]
I am of course fine with you, using whatever tools you like!
Do I just install the changelogset-bot in the intlify organization?

**Comment by ota-meshi** [comment]
> Do I just install the changelogset-bot in the intlify organization?

The bot will send comments to the PR, so I think it's a good idea to install it only in the repositories we use.
I installed it.

**Comment by kazupon** [comment]
Okay,  Let's ship with the tools you are comfortable with!

**Pull request merged**